Assets/Scripts/Models/Inventory.cs

```diff
@@ -191,4 +159,13 @@ private void ImportPrototypeSettings(int defaulMaxStackSize, float defaultBasePr
             Category = defaultCategory;
         }
     }
+
+    private void OnStackSizeChanged(Inventory inventory)
```

Comment:
Ok. It feels a bit weird to me, since you are not using it to attach it to an event and having it work when something changes. One thing should be to trigger/send an event and other to receive it. But if we go for the standard, that is fine.